ngi_pipeline/engines/piper_ngi/launchers.py

```diff
@@ -216,7 +216,7 @@ def sbatch_piper_sample(command_line_list, workflow_name, project, sample,
         raise RuntimeError('No SLURM project id specified in configuration file '
                            'for job "{}"'.format(job_identifier))
     slurm_queue = config.get("slurm", {}).get("queue") or "node"
-    num_cores = config.get("slurm", {}).get("cores") or 16
+    num_cores = config.get("slurm", {}).get("cores") or 8
```

Comment:
Yeah it should be but if the user doesn't specify in the config file we default to 8